xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -330,6 +332,7 @@ public void ldsResourceUpdated() {
 
   @Test
   public void ldsResourceUpdate_withFaultInjection() {
+    Assume.assumeTrue(useProtocolV3());
```

Comment:
Why this?

Reply:
It does not make sense to test `ldsResourceUpdate_withFaultInjection ` with `ClientXdsClientV2Test`, the assertions are not true for v2.